src/app/gateways/views/DelegatedGatewayDetailView.vue

```diff
@@ -18,19 +18,12 @@
       }"
     >
       <AppView>
-        <DataSource
+        <DataLoader
```

Comment:
Same as above, mostly (but there will be exceptions) you only want one DataLoader on a page and if there are multiple HTTP requests to be made, use non-blocking DataSources to feed the data into the single blocking DataLoader

Reply:
I changed this back, too, now.